index.html

```diff
@@ -27,15 +27,17 @@ <h1>Medicina Emocional <br> para tu vida</h1>
     <section id="Fan-de-las-Flores">
         <div class="container">
             <div class="img-container"></div>
-            <h2>Somos <span class="color-acento"> Fanes de las Flores!</span></h2>
-            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quas molestias autem, porro ratione dolorem qui sit repellendus odio assumenda sint excepturi itaque blanditiis officiis, aperiam quibusdam, unde fuga ducimus eius.</p>
+                <div class="texto">
+                <h2>Somos <span class="color-acento"><br> Fanes de las Flores!</span></h2>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quas molestias autem, porro ratione dolorem qui sit repellendus odio assumenda sint excepturi itaque blanditiis officiis, aperiam quibusdam, unde fuga ducimus eius.</p>
+            </div>
         </div>   
     </section>
 
-    <section id="Esencias">
+    <section id="esencias">
         <div class="container">  
             <h2>Las Vibraciones y Las Esencias</h2>
-            <div class="esencias">
+            <div class="textoesencias">
```

Comment:
usualmente las clases CSS cuando están compuestas de dos palabras se dividen por guiones ej: **texto-esencias** cambia las clases con ese formato.

Reply:
This adjusted is solved in next commit of branch feature/body